### PR TITLE
updates and working maven publication

### PR DESCRIPTION
**Pull request by NikkyAI**
mod version should match the kotlin version

builds fine locally

### CODE REVIEW

**Comment by NikkyAI** [comment]
current output when trying to run a sample project with the language adapter in multimc
https://paste.ee/p/yzcjK

**Comment by NikkyAI** [comment]
after readding the shadow plugin this error happens again:
```
org.gradle.api.InvalidUserDataException: Cannot change dependencies of configuration ':compile' after it has been included in dependency resolution.
```
so it seems directly related to shadowjar

**Comment by NikkyAI** [comment]
managed to load a kotlin mod in multimc with the unshaded version of `fabric-language.kotlin` in localMaven
and adding these libraries to minecraft

```
        {
            "name": "net.fabricmc:fabric-language-kotlin:1.3.0",
            "url": "http://localhost/repository/"
        },
        {
            "name": "org.jetbrains.kotlin:kotlin-stdlib:1.3.0",
            "url": "https://jcenter.bintray.com/"
        },
        {
            "name": "org.jetbrains.kotlin:kotlin-reflect:1.3.0",
            "url": "https://jcenter.bintray.com/"
        },
        {
            "name": "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.0",
            "url": "https://jcenter.bintray.com/"
        },
```

**Comment by NikkyAI** [comment]
depends on https://github.com/FabricMC/fabric-loom/pull/19
also fixes #3

File: src/main/kotlin/net/fabricmc/language/kotlin/KotlinLanguageAdapter.kt [hunk 1]
```diff
@@ -16,24 +16,15 @@
...
-            val instanceField = clazz.getField("INSTANCE")
-            val instance = instanceField.get(null) ?: throw NullPointerException()
-            logger.debug("Found INSTANCE field for ${clazz.name}")
-            instance
-        } catch (e: Exception) {
-            logger.debug("Unable to find INSTANCE field for ${clazz.name}, constructing new instance")
-            clazz.newInstance()
-        }
+    override fun createInstance(clazz: Class<*>): Any = clazz.kotlin.objectInstance ?: run {
+        logger.debug("Unable to find INSTANCE field for ${clazz.name}, constructing new instance")
```
Comment (InsomniaKitten) [inline src/main/kotlin/net/fabricmc/language/kotlin/KotlinLanguageAdapter.kt]: This should be `Kotlin object instance`, not `INSTANCE field`

Comment (NikkyAI) [reply src/main/kotlin/net/fabricmc/language/kotlin/KotlinLanguageAdapter.kt]: fix

**Pull request merged**